cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -43,6 +57,10 @@ VhpiSignalObjHdl::~VhpiSignalObjHdl()
 
     if (m_binvalue.value.str)
         free(m_binvalue.value.str);
+
+    LOG_DEBUG("Releasing VhpiSignalObjHdl handle at 0x%X\n", get_handle<vhpiHandleT>());
```

Comment:
Does this do printf formatting? If so, you need `zX`.

Reply:
Changed to `%p` and cast handle to `(void *)`.